tests/test_check_docker.py

```diff
@@ -728,7 +730,7 @@ def test_print_results(check_docker, capsys, messages, perf_data, expected):
     assert out.strip() == expected
 
 
-@pytest.mark.parametrize("messages, perf_data, no_ok, no_performance, expected", (
+@pytest.mark.parametrize("messages, perf_data, no_ok, no_unknown, no_performance, expected", (
```

Comment:
You added a test parameter but you didn't update the data to match it. Have you used parameterized pytest tests before? It is pretty simple (and also a little ugly).
The string is a ordered list of names for all of the values in each of the tuples that follow. Each tuple in the list describes the setup of a single test case. For example

``` @pytest.mark.parametrize("one, two", ((1,2),(11,22)))
def test_func(one, two) 
    ...
```

This executes does something like this
```
for params in ((1,2),(11,22)):
  one, two = params
  test_func(one=one, two=two)
```

Can you add the missing entries to the tuples and also add tuples that exercise the code you are changing.

Don't worry if you break something, I can help you out. Also my dev documentation should help you get the test running on your machine.